releasenotes/notes/0.9/new-features-0.9-159645f977a139f7.yaml

```diff
@@ -64,7 +64,7 @@ features:
     passed a set of kwargs on each call with the state of the pass maanger after
     each pass execution. Currently these kwargs are:
 
-     * pass_ (Pass): the pass being run
```

Comment:
The parameter name is actually `pass_` (https://github.com/Qiskit/qiskit-terra/blob/master/qiskit/transpiler/passmanager.py#L184 ) because `pass` is a reserved word in python. We might want to escape the character here if it's causing a build error.